### ISSUE TITLE
Improve docs: A11y: Link vs text contrast does not meet WCAG guidelines

### ISSUE THREAD

**Issue by mobot11** [SAP/fundamental-react#460]
### Description
WCAG requires "A 3:1 contrast between the link text color and the surrounding non-link text color." This might turn into an issue with fiori-fundamentals. Can check the allowable contrast levels here: https://webaim.org/resources/linkcontrastchecker/

### Versions
**fundamental-react:**
**fiori-fundamentals:**

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._

**Comment by bcullman** [comment]
@mobot11 3:1 contrast ratio is reserved for exceptionally large text. For the majority of the text, we actually need to hit 4.5:1.

**Comment by droshev** [comment]
@mobot11 is this specifically to fundamental-react or it comes from fiori-fundamentals?

**Comment by mobot11** [comment]
@droshev Looks like these are the defaults from fiori-fundamentals. I can open an issue on that repo as well.

**Comment by jbadan** [comment]
@mobot11 @droshev A couple of the buttons have this same problem, specifically medium (orange) and light buttons.

**Comment by mobot11** [comment]
@bcullman I _believe_ specifically for the contrast between the link color and the text color we are only required to provide a 3:1 ratio according to https://webaim.org/articles/contrast/#only.

**Comment by gszalay** [comment]
The short version: I don't see any problems.

I reviewed the Fundamental Link (https://sap.github.io/fundamental/components/link.html) page and had 2 quick findings:

1. The colors in Fundamental do not match the colors in the Fiori 3 Master Sketch file. I would therefore assume the Fundamental colors are not final and will change. 
2. This section of WebAim (https://webaim.org/articles/contrast/#only) applies to links when no other indicator (such as underline) is used. So, the 3:1 rule is not applicable if there is an underline (which is displayed in Fundamental)
